javascript/modules/client/src/client.ts

```diff
@@ -565,20 +565,21 @@ export class Client extends ClientCore implements IClient {
     } else if (!params.tokenAddress) {
       throw new NoTokenAddress();
     }
-    // resolve ens ?
-    // let daoAddreess = params.daoAddress;
-    // if (!isAddress(daoAddreess)) {
-    //   await this.web3.ensureOnline();
-    //   const provider = this.web3.getProvider();
-    //   if (!provider) {
-    //     throw new NoProviderError();
-    //   }
-    //   const resolvedAddress = await provider.resolveName(daoAddreess);
-    //   if (!resolvedAddress) {
-    //     throw new InvalidAddressOrEnsError();
-    //   }
-    //   daoAddreess = resolvedAddress;
-    // }
+    
+    // resolve ens
+    let daoAddreess = params.daoAddress;
```

Comment:
If we leave this piece of code, shouldn't `daoAddress` be used below this block?